app/domain/authentication/authn_jwt/signing_key/fetch_jwks_uri_signing_key.rb

```diff
@@ -57,11 +59,34 @@ def jwks_uri_secret
           )[JWKS_URI_RESOURCE_NAME]
         end
 
+        def net_http_start(host, port, use_ssl, &block)
+          if @ca_cert
+            @http_lib.start(
+              host,
+              port,
+              use_ssl: use_ssl,
```

Comment:
What if ca_cert is not nil and use_ssl is nil or the opposite , maybe we will want to throw an error in that case?

Reply:
`use_ssl` cannot be `nil` it's boolean and it's private
If the value is `false` means the uri is `http` the `cert_store` parameter is ignored.
In any case it's a good point to throw an error in case of such misconfiguration.